src/config.ts

```diff
@@ -58,7 +58,10 @@ export async function loadConfig(configFilename: string = CONFIG_FILENAME): Prom
 
   log('Resolved config file.');
 
-  const loaded = mergeDeepRight(DEFAULT_CONFIG, loadedConfig) as Configuration;
+  const loaded = (mergeDeepRight<Configuration, Configuration>(
+    DEFAULT_CONFIG,
+    loadedConfig
+  ) as unknown) as Configuration;
```

Comment:
.

Reply:
If there's a problem here, we should use `lodash.merge` instead of Ramda. We don't need the whole ramda thing for such a tiny functionality.

Reply:
Updating ramda to `v0.29.0` solved the type issue. Bumping the ramda version